### PR TITLE
Better retries

### PR DESCRIPTION
**Pull request by rowleya**
Poorly named branch, but this increases the default number of retries to 10, except for the initial get_version during boot which is 3 (as otherwise boot is really slow).

### CODE REVIEW

File: spinnman/transceiver.py [hunk 1]
```diff
@@ -1077,7 +1077,7 @@ def _try_to_find_scamp_and_boot(
         current_tries_to_go = tries_to_go
         while version_info is None and current_tries_to_go > 0:
             try:
-                version_info = self.get_scamp_version()
+                version_info = self.get_scamp_version(n_retries=3)
```
Comment (dkfellows) [inline spinnman/transceiver.py]: Why is this place 3 when nowhere else is?

Comment (dkfellows) [reply spinnman/transceiver.py]: Oh, because it's for boot.

Comment (andrewgait) [reply spinnman/transceiver.py]: I'm beginning to sound like @alan-stokes but perhaps this should be a constant called BOOT_RETRIES?

Comment (rowleya) [reply spinnman/transceiver.py]: Yes, it probably should...

Comment (rowleya) [reply spinnman/transceiver.py]: I have added this constant now...

**Pull request merged**